docs/proxy.md

```diff
@@ -7,21 +7,61 @@ paths:
     - proxy
 ---
 
-# [](./proxy)Proxy
+# [](./proxy) Proxy
 
-Apify Proxy provides access to Apify's proxy services that can be used in [actors]({{@link actors.md}})
-or any other application that support HTTP proxies.
-Currently, Apify Proxy provides access to [datacenter proxy servers]({{@link proxy/datacenter_proxy.md}}),
-[residential proxy]({{@link proxy/residential_proxy.md}}),
-and [Google SERP proxy]({{@link proxy/google_serp_proxy.md}}).
-It supports HTTP as well as other protocols like HTTPS and FTP.
+[Apify Proxy](https://apify.com/proxy) allows you to disguise yourself when web scraping to reduce your chance of being [blocked]({{@link web_scraping_101/anti_scraping_techniques.md#ip-address-based-blocking}}).
 
-You can view your Apify Proxy settings on the [Proxy page](https://my.apify.com/proxy) in the app.
+You can use proxies in your [actors]({{@link actors.md}}) or any other application that supports HTTP, HTTPS and FTP protocols. Apify Proxy monitors the health of your IP pool and intelligently [rotates addresses](#ip-address-rotation) to prevent detection.
 
+**You can view your proxy settings and password on the [Proxy](https://my.apify.com/proxy) page in the Apify app.**
 
-*   [**Connection settings**]({{@link proxy/connection_settings.md}})
-*   [**Datacenter proxy**]({{@link proxy/datacenter_proxy.md}})
-*   [**Residential proxy**]({{@link proxy/residential_proxy.md}})
-*   [**Google SERP proxy**]({{@link proxy/google_serp_proxy.md}})
-*   [**Troubleshooting**]({{@link proxy/troubleshooting.md}})
+## [](#our-proxies) Our proxies
 
+[Datacenter proxy]({{@link proxy/datacenter_proxy.md}}) – the fastest and cheapest option, it uses data centers to mask your IP address. Chance of blocking due to other users' activity. [[Code examples]({{@link proxy/datacenter_proxy/examples.md}})]
+
+[Residential proxy]({{@link proxy/residential_proxy.md}}) – IP addresses located in homes and offices around the world. These IPs have the lowest chance of blocking. [[Code examples]({{@link proxy/residential_proxy.md#connecting-to-residential-proxy}})]
```

Comment:
The menu on the left and this link are both missing the actual examples of usage of residential proxy. I do not think this is important, but the inconsistency bothered me a bit :)

Reply:
@mnmkng brought up that there's a lot of duplication if we have a separate examples page for residential, since only the 'groups' property changes.

Let's try the minimalist approach, and if it doesn't work (we can find out once we get the feedback widget), then we can add the residential examples back in :)